forward_engineering/utils/indexes.js

```diff
@@ -0,0 +1,30 @@
+/**
+ *
+ * @param {{index: object, keyIdToNameMap: object}} param
+ * @returns {object}
+ */
+const injectKeysNamesIntoIndexKeys = ({ index, keyIdToName = {} }) => ({
+	...index,
```

Comment:
I would extract these inline spread statements as variables to make the code a bit more readable.